packages/fx-core/src/plugins/resource/bot/constants.ts

```diff
@@ -146,6 +146,11 @@ export class Retry {
   public static readonly BACKOFF_TIME_MS = 5000;
 }
 
+export class DeployStatus {
+  public static readonly RETRY_TIMES = 60;
```

Comment:
According your experienced behavior, the longest deploy time is 8 min, so better to have more buffer here - let the whole retry be 2*8 = 16 min.

Reply:
Or, is there any hard timeout line from Kudu?